.github/workflows/ci_tests.yaml

```diff
@@ -0,0 +1,108 @@
+# This workflow will install Python dependencies, run tests and lint with a single version of Python
+# For more information see: https://help.github.com/actions/language-and-framework-guides/using-python-with-github-actions
+
+name: Tests
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+  # Schedule daily tests
+  schedule:
+    - cron: '0 0 * * *'
+
+jobs:
+  test:
+    name: ${{ matrix.os }} - Python ${{ matrix.python-version }}
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        python-version: [3.7]
```

Comment:
Should we expand this to cover 3.6 and 3.8 as well?

Reply:
Yes, but after we confirm that the current jobs all succees.

Reply:
Maybe I should revert 1b947d3fde42cbe99988cf341f26e88418399640 so that GH Actions runs on GMT 6.0, otherwise they will definitely fail.

Reply:
Yes, 6.0.0 first.

Reply:
Ok, done. Should we disable the Azure Pipelines tests? Maybe just leave the "Style Checks" one so we'll know when it's working again.

Reply:
Yes, sounds good.

Reply:
FYI, I've changed the "Required" status checks from Azure Pipeline to Github Actions for the Windows and macOS builds.